spec/meilisearch_spec.rb

```diff
@@ -13,9 +13,19 @@
   end
 
   it 'allows to set a custom timeout and max_retries' do
-    client = MeiliSearch::Client.new($URL, $MASTER_KEY, timeout: 0, max_retries: 2)
-    expect do
-      client.indexes
-    end.to raise_error(Timeout::Error)
+    client = MeiliSearch::Client.new('http://127.0.0.1:8800', 'masterKey', timeout: 20, max_retries: 2)
```

Comment:
```suggestion
    client = MeiliSearch::Client.new($URL, 'masterKey', timeout: 20, max_retries: 2)
```

I think this may be the cause of the failing test

```
  1) MeiliSearch allows to set a custom timeout and max_retries
     Failure/Error: expect(client.healthy?).to be true

       expected true
            got false
     # ./spec/meilisearch_spec.rb:17:in `block (2 levels) in <top (required)>'
```